### DISCUSSION TITLE
saf-training/courses/profile-dev-test/03

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#74] [Ideas]
# saf-training/courses/profile-dev-test/03

Required Software RVM, or another Ruby Management Tool; Ruby v3 or higher; Git; VS Code or another IDE; Docker (if you want to test hardened and non-hardened containers); AWS CL...

https://mitre.github.io/saf-training/courses/profile-dev-test/03.html

**Comment by seanlongcc** [comment]
- While progressing through the environment setup, I noticed that it
   doesn’t include a guide or links to installing Ruby, InSpec, and
   Kitchen. Having a guide for setting up a Platform One account and P1 Harbor
   token would also be helpful. 
 - Maybe have an “Optional Accounts” as the AWS console account doesn't seem  
   necessary if not using AWS.
 - In the Post-Setup checks section, the `docker pull   
   https://repo1.dso.mil/dsop/redhat/ubi/ubi8` command
   does not work.
	   -	removing the `https://` makes it work, but then gives a new error `Error response from daemon: Head "https://repo1.dso.mil/v2/dsop/redhat/ubi/ubi8/manifests/latest": error parsing HTTP 404 response body: no error details found in HTTP response body: "{\"message\":\"dependency proxy not enabled\",\"status\":\"error\",\"http_status\":404}"`
